package.json

```diff
@@ -66,6 +66,9 @@
     "webpack": "^1.9.6",
     "webpack-dev-server": "^1.8.2"
   },
+  "dependencies": {
+    "lodash": "~4.0.0"
```

Comment:
Using the `~` because we're referencing internal modules. We try to avoid issues with internal modules for the `~` range.